tools/tests/com.liferay.ide.project.ui.tests/src/com/liferay/ide/project/ui/tests/SWTBotTestBase.java

```diff
@@ -50,4 +50,9 @@ public static void closeWelcomeSwitchToLiferay()
         {
         }
     }
+    
+    protected String getBundleId()
```

Comment:
We don't need a BUNDLE_ID in the SWTBotBase class.  The bundle ids should only be in the sub-classes.